symplyphysics/laws/thermodynamics/relative_humidity.py

```diff
@@ -1,32 +1,31 @@
-from sympy import (Eq, solve, S)
+from sympy import (Eq, solve)
 from symplyphysics import (
     units,
     Quantity,
     Symbol,
     print_expression,
     validate_input,
     validate_output,
-    dimensionless,
-    convert_to
+    dimensionless
 )
 
 # Description
 ## Relative humidity is the ratio of water vapor pressure p to saturated vapor pressure p0 at a given temperature.
+## Saturated vapor is vapor in dynamic equilibrium with a liquid or solid of the same composition in a closed system.
+## In such a pair, the number of evaporating molecules is equal to the number of condensing molecules per unit of time.
+## Water vapor is the gaseous aggregate state of water.
 
 ## Law is: ph = (p / p_sat) * 100, where
 ## ph - relative humidity,
 ## p - water vapor pressure,
```

Comment:
It seems it is not just some random water vapor, but the vapor we are measuring humidity for. This is not really clear from your description.
Mention that relative humidity is for this vapor. And I see somewhere it is called `actual vapor pressure`.